src/analysis/modals/ExportAnalysisModal/useAnalysisExportState.test.ts

```diff
@@ -39,14 +39,13 @@ jest.mock(
   })
 );
 
-type WorkspaceUtilsExports = typeof import('src/workspaces/common/state/useWorkspaces');
-jest.mock(
-  'src/workspaces/common/state/useWorkspaces',
-  (): WorkspaceUtilsExports => ({
-    ...jest.requireActual('src/workspaces/common/state/useWorkspaces'),
+type useWorkspacesExports = typeof import('src/workspaces/common/state/useWorkspaces');
```

Comment:
Super nit: types are usually capitalized.

```suggestion
type UseWorkspacesExports = typeof import('src/workspaces/common/state/useWorkspaces');
```

Reply:
Will do.

Reply:
Changed all of the lowercase types.